src/pages/Product/Product.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './Product.scss';
+import ProTab from './Protab';
+import ProHead from './Prohead';
+import ProList from './Prolist';
+
+class Product extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      prolist: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('data/ListItemData.json')
```

Comment:
```suggestion
    fetch('/data/ListItemData.json')
```
절대경로 붙여주세요 안그럴경우 현재 위치한 url 기준으로 계산하기 때문에 오동작 할 여지가 있습니다.